arch-travis.sh

```diff
@@ -42,11 +42,21 @@ configure_volumes() (
     while read -r vol; do printf -- '-v "%s"\n' "$vol"; done<<<"${volumes[*]}"
 )
 
+# regression test for deprecated arch-travis configuration scheme.
+{
+    if travis_yml arch script >/dev/null 2>&1; then
+      echo '*** WARNING! Your current arch-travis configuration is deprecated'
```

Comment:
This is more than deprecated as the old key won't work anymore. I think this is ok, we should just rename it to outdated or so.

Reply:
updated :smirk: